lib/jsdom/level2/style.js

```diff
@@ -19,7 +19,9 @@ exports.addToCore = core => {
   // - Load events are not in the specs, but apparently some browsers
   //   implement something. Should onload only fire after all @imports have been
   //   loaded, or only the primary sheet?
-
+  cssom.setup({
+    globalObject: core._globalObject
+  });
```

Comment:
I don't understand this design. cssom is a single module, shared across the entire Node.js process. Whereas each jsdom instance has a separate `_globalObject`.

From what I can tell, this will cause CSSOM's `globalObject` reference to get overridden if someone creates two jsdom instances, e.g., via

```js
const dom1 = new JSDOM();
const dom2 = new JSDOM();
```

That seems like it'll be problematic.

Reply:
The module needs to be able to create one copy of CSSStyleSheet, etc., per jsdom instance, via a factory function that returns them.